src/draw/sw/lv_draw_sw_gradient.h

```diff
@@ -25,6 +25,12 @@ extern "C" {
 #error LVGL needs at least 2 stops for gradients. Please increase the LV_GRADIENT_MAX_STOPS
 #endif
 
+#define LV_GRAD_LEFT    LV_PCT(0)
+#define LV_GRAD_RIGHT   LV_PCT(100)
+#define LV_GRAD_TOP     LV_PCT(0)
+#define LV_GRAD_BOTTOM  LV_PCT(100)
+#define LV_GRAD_CENTER  LV_PCT(50)
```

Comment:
IMO these are not required. Using `lv_pct()` directly in the examples better expresses what is happening and it's more informative for the users too.

Reply:
This was modeled after the CSS keywords.

Reply:
Okay, let's keep them.